src/dev-app/index.html

```diff
@@ -30,6 +30,7 @@
   <script src="system-config.js"></script>
   <script src="https://www.youtube.com/iframe_api"></script>
   <script src="https://maps.googleapis.com/maps/api/js"></script>
+  <script src="https://unpkg.com/@googlemaps/markerclustererplus/dist/index.min.js"></script>
```

Comment:
Should we mention somewhere in the docs that this script is necessary?

Reply:
Yeah, good catch. Adding that now.